### ISSUE TITLE
Removal criteria

### ISSUE THREAD

**Issue by davisonio** [bevry/staticsitegenerators-list#459]
Should we change the criteria for something showing up on the list?

I think the main value gained from this static site generator list is for someone to see a list of *working* static site generators if they'd like to try out a few.

We have the defunct setting but I believe it still shows up on the website the same, maybe we should have these defunct listings not show up on the website or at least be visible that they likely no longer work.

Quite a few listed site generators no longer work.

**Comment by balupton** [comment]
> We have the defunct setting but I believe it still shows up on the website the same, maybe we should have these defunct listings not show up on the website or at least be visible that they likely no longer work.

Good idea. I'll be updating the website this week or next, so I'll make the defunct ones strikethrough.
